microsoft-support.md

```diff
@@ -13,8 +13,8 @@ You can also request community support on GitHub (for free), but there is no gua
 
 Support is conditional on using the latest .NET patch update and a supported operating system, as defined by:
 
+* [.NET releases](releases.md)
 * [.NET release policies](release-policies.md)
-* [.NET release lifecycle](releases.md)
 * [Operating system lifecycle](os-lifecycle-policy.md).
```

Comment:
Should we rename to: ".NET Supported Operating System lifecycle" to follow the pattern of the bullets listed above. Plus this is more descriptive

Reply:
Done.